### PR TITLE
minor: Change encoding to UTF-8 in `open()` calls.

### PR DESCRIPTION
**Pull request by ganpa3**
This allows zulint to run on Windows.

### CODE REVIEW

File: example-lint [hunk 1]
```diff
@@ -27,7 +28,8 @@ def run() -> None:
     ]  # type: List[str]
     by_lang = linter_config.list_files(file_types, exclude=EXCLUDED_FILES)
 
-    linter_config.external_linter('mypy', ['tools/run-mypy'], ['py'], pass_targets=False,
+    mypy_command = ['python', 'tools\\run-mypy'] if sys.platform == 'win32' else ['tools/run-mypy']
```
Comment (andersk) [inline example-lint]: Shouldn’t `[sys.executable, 'tools/run-mypy']` do the right thing on both platforms?

This should either be a separate commit or the commit message generalized.

**Comment by timabbott** [comment]
This looks great implementation-wise, but @ganpa3 can you expand the commit messages to provide more of the context?  "This makes it work on Windows" is helpful, but it should explain what the actual issue was on Windows with the previous version.

**Comment by ganpa3** [comment]
@timabbott Updated the commit message.

**Comment by timabbott** [comment]
Merged, thanks @ganpa3!
